docs/wallet/quick-start-webwallet.md

```diff
@@ -42,6 +42,22 @@ There are 3 methods to connect with the Web Wallet. Feel free to use the one you
 
 ![](/img/webwallet/config-02.png)
 
+## Ledger
+
+If you want to sign transactions with a Ledger hardware wallet, choose `Browser Wallet` as your connection option. This will prompt a popup to verify that you're connected to Ledger. Connect the device and follow the required steps, then click `YES`.
+
+![](/img/ledger/01.png)
+
+Make sure to allow contract data in transactions in your Ethereum application and keep that application opened as follows:
+
+![](/img/ledger/02.png)
+
+To confirm your actions or change values in the settings, press both of the buttons with your fingers as shown above.
+
+Lastly, please check the following:
+1. Your MetaMask is connected to Ledger. Otherwise, you will get an unauthorized spend.
```

Comment:
so these 2 things are now checked automatically by the wallet (and will give you an early warning and not allow you to continue if they are not set correctly). so we dont have to include it here, but it also doesn't hurt. i'll leave it up to you.

Reply:
I'll leave it just in case : )